packages/design-system/src/components/Status/variations/StatusInProgress.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import i18n from 'i18next';
+import StatusPrimitive, { StatusProps } from '../Primitive/StatusPrimitive';
+
+export type StatusInProgressProps = Omit<StatusProps, 'icon' | 'variant' | 'inProgress'>;
+
+const StatusInProgress = React.forwardRef(
+	(props: StatusInProgressProps, ref: React.Ref<HTMLSpanElement>) => {
+		return (
+			<StatusPrimitive inProgress variant="inProgress" {...props} ref={ref}>
```

Comment:
Wdyt about having `variants` as an enum to be able to use it in each component.

Reply:
We tried it, but it seems that TS then requires you to use said enum in your implem. It meant that for people who wanted to use `Status`, they would have had to import both `Status` and `StatusVariantEnum` in order to do something like `<Status variant={StatusVariantEnum.warning} />` which seemed tedious... 

Eventually here the typing is as strong (through keyof typeof the object) and the prop autocompletes fairly well. Is that an acceptable tradeoff to you?